android/app/src/main/java/org/fossasia/openevent/adapters/viewholders/SpeakerViewHolder.java

```diff
@@ -50,9 +54,35 @@ public SpeakerViewHolder(View itemView, Context context) {
         //Attach onClickListener for ViewHolder
         itemView.setOnClickListener(view -> {
             String speakerName = speaker.getName();
+                       
+            Bundle bundle = null;
+            
             Intent intent = new Intent(this.context, SpeakerDetailsActivity.class);
             intent.putExtra(Speaker.SPEAKER, speakerName);
-            this.context.startActivity(intent);
+            
+            if(this.context instanceof Activity){
+                
+                    if(android.os.Build.VERSION.SDK_INT >= android.os.Build.VERSION_CODES.LOLLIPOP){
+
+                       bundle = ActivityOptions.makeSceneTransitionAnimation((Activity) this.context, speakerImage, speakerImage.getTransitionName()).toBundle();
+                       this.context.startActivity(intent, bundle);
+
+                      }
+
+                    else{
+
+                        this.context.startActivity(intent);
+
+                    }
+            }
+            
+            else{
+       
+             this.context.startActivity(intent);
+
+             }
```

Comment:
The indentation of whole block is off and there is lot of white space. Please fix